src/app/terms/page.js

```diff
@@ -0,0 +1,94 @@
+"use client";
+/* eslint-disable react/no-unescaped-entities */
+
+import { Box, Typography } from '@mui/material';
+
+
+
+export default function TermsAndConditionsPage() {
+  return (
+    <Box sx={{
+      fontFamily: 'Manrope, sans-serif',
+      backgroundColor: '#F2F2F2',
+      p: '40px 20px',
```

Comment:
Same for p, we do not use specific values. Use integer values vs '20px'.
Did you read MUI docs on p usage at all?